kubernetes-client/src/com/goyeau/kubernetes/client/KubernetesClient.scala

```diff
@@ -61,6 +63,24 @@ class KubernetesClient[F[_]: Async: Logger](
       encoder: Encoder[CustomResource[A, B]],
       decoder: Decoder[CustomResource[A, B]]
   ) = new CustomResourcesApi[F, A, B](httpClient, config, authorization, context)
+
+  def customRequest(
+      request: Request[F]
+  ): F[Request[F]] =
+    Request[F](
+      method = request.method,
+      uri = config.server.resolve(request.uri),
+      httpVersion = request.httpVersion,
+      headers = request.headers,
+      body = request.body,
+      attributes = request.attributes
+    ).withOptionalAuthorization(authorization)
+
+  def customRequest(request: WSRequest): F[WSRequest] =
+    request
+      .copy(uri = config.server.resolve(request.uri))
+      .withOptionalAuthorization(authorization)
+
```

Comment:
Sounds like this is unrelated to the intention of this PR: pod log endpoint ?

Reply:
I extracted the custom requests stuff into a separate PR: https://github.com/joan38/kubernetes-client/pull/246